File: CursoProgramacionCero/Algoritmos/año_bisiesto.js
// Algoritmo de año bisiesto
let year;

year = 2023;

if ((year % 4 == 0 && year % 100 != 0) || year % 400 == 0) {
    console.log(year + " es un año bisiesto");
} else {
    console.log(year + " no es un año bisiesto");
}

File: CursoProgramacionCero/Algoritmos/año_bisiesto.py
# Algoritmo de año bisiesto
año = 0

print ("Por favor ingresa un año");

año = int(input())

if (año % 4 == 0 and año % 100 != 0) or año % 400 == 0:
    print (f"{año} es un año bisiesto")
else:
    print (f"{año} no es un año bisiesto")